docs/usage/security-and-permissions.md

```diff
@@ -0,0 +1,56 @@
+# Security and Permissions
+
+## Comparison table
+
+Refer to this table for a overview of the permissions needed for the different ways to run Renovate:
+
+| Permission name   | Renovate hosted app |  Forking Renovate  |
+| ----------------- | :-----------------: | :----------------: |
+| Dependabot alerts |       `read`        |   not applicable   |
```

Comment:
I think the full fat Renovate app needs `Dependabot alerts` `read` access to be able to provide security update PRs?

Forking Renovate doesn't have `read` access to the Dependenda bot security alerts. Does this mean that Forking Renovate doesn't raise security PRs?

If so, should I go edit the docs for Forking Renovate to mention this drawback?

https://docs.renovatebot.com/getting-started/running/#forking-renovate-app

Reply:
I think we should maybe add the permissions

Reply:
If I were running Forking Renovate, I would expect it to be able to open security update PRs, after all that's what I'm using the update dependencies bot for in the first place. So I think it would make sense to add the permission/feature.

In the end, it's your app, so your choice. 😉

Reply:
Yes, I think it's just an oversight